src/js/view/controls/controlbar.js

```diff
@@ -439,8 +439,12 @@ export default class Controlbar {
         this.elements.alt.textContent = altText;
     }
 
-    // Close menus if it has no event.  Otherwise close all but the event's target.
+    // Close menus if it has no event.
     closeMenus(evt) {
+        if (!this.menus) {
+            return;
```

Comment:
No event or no menus?

Reply:
@waxidiotic @boyntoni,

@pajong noticed that `this.menus` is never being set because or recent changes. It always should be set so adding this guard should not have been added because it doesn't address the regression and instead adds code which causes us all to ignore it.

Reply:
@robwalch I'll remove it in another PR.

Reply:
Actually, now that I think of it - maybe @pajong should remove it in #3215.

Reply:
It is done.